### PR TITLE
Fixed the wrong API call being made when customer wants to fetch all deployments

### PR DESCRIPTION
**Pull request by rujhan-arora-astronomer**
## Description

Fixed the wrong API call being made when a customer wants to fetch all deployments.
When a customer wants to fetch all deployments, the cli right now makes a call to an API which requires workspace ID. So, I am using another api instead which should return all the deployments.

## 🎟 Issue(s)

[Related #6602
](https://github.com/astronomer/issues/issues/6602)

## 🧪 Functional Testing

Run astro deployment list --all. It should list all the deployments irrespective of workspace.

## 📸 Screenshots

Screenshot 2024-11-19 at 7 45 14 PM
Screenshot 2024-11-19 at 7 45 05 PM

## 📋 Checklist

- [x] Rebased from the main (or release if patching) branch (before testing)
- [x] Ran `make test` before taking out of draft
- [x] Ran `make lint` before taking out of draft
- [x] Added/updated applicable tests
- [ ] Tested against [Astro-API](https://github.com/astronomer/astro/) (if necessary).
- [x] Tested against [Houston-API](https://github.com/astronomer/houston-api/) and [Astronomer](https://github.com/astronomer/astronomer/) (if necessary).
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: houston/deployment.go [hunk 1]
```diff
@@ -233,6 +239,38 @@ var (
 		},
 	}
 
+	PaginatedDeploymentsGetRequest = queryList{
```
Comment (jeremybeard) [inline houston/deployment.go:242]: This is mixing spaces and tabs, can we just use tabs? Looks like the other requests use tabs.

**Review by jeremybeard — APPROVED** [review]
One nit comment but approving ahead of that

File: houston/deployment.go (hunk 1 above)
Comment (rujhan-arora-astronomer) [reply houston/deployment.go:242]: Done!

**Comment by rujhan-arora-astronomer** [comment]
@jeremybeard Addressed the nitpick. Can you merge this too plz, thank you!

**Review by neel-astro — APPROVED** [review]

**Pull request merged**